Main_Page/templates/Main_Page/base.html

```diff
@@ -63,6 +64,40 @@
   color: rgb(29,161,242,1.00);
 }
 
+/* POPUP Information */
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0,0,0,0.4);
+}
+
+.modal-content {
+  background-color: black;
+  margin: 15% auto;
+  padding: 20px;
+  border: 1px solid rgb(29,161,242,1.00);
+  width: 80%;
+}
+
+.close {
+  color: rgb(29,161,242,1.00);
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.close:hover {
+  color: #aaa;
+  text-decoration: none;
+  cursor: pointer;
+}
+
```

Comment:
Is any of this CSS used anywhere in this branch?

Reply:
Not yet, this CSS is setup for the popup window for the login and register